### ISSUE TITLE
Spatial Audio

### ISSUE THREAD

**Issue by ukting** [LNSSPsd/PodsGrant#52]
Any plans to work on spatial audio? Seems like adaptive transparency works for me on iOS 15.4.1 but spatial audio doesn’t seem to track properly vs an iOS 16 device I have, I know this might not be fixable but curious if you were working on it.

**Comment by LNSSPsd** [comment]
Adaptive transparency works because it is a settable bluetooth characteristic of AirPods Pro, but the spatial audio is more likely to be dependent to the connected phone instead of the earbuds. I could be wrong, and I will check on this later to see if I am.

**Comment by ukting** [comment]
You’re probably right and I’m making my observations based on 1st gen airpod pros on iOS 15.4.1, the spatial audio worked on that but that firmware was supported at that time.  Spatial audio has been available on my device(IP13 PM) so I’m not sure how that will work since the firmware isn’t supported anyway.

either way great work on the tweak to even allow us some native like compatibility!

**Comment by rastafaa** [comment]
Yeah spatial audio doesn’t work right on 14.8 either, and it works on my 1st gen pros. It gets enabled fine but doesn’t track with head / phone (same as without the tweak)

another thing that still doesn’t work is the optimized charging feature. But most other features are working at least.

**Comment by ukting** [comment]
Yeah same behavior here, I was on 14.4 until last month on an older device and I had no spatial audio until recently after my new device on 15.4.1 and now I just got my APP2 and no spatial again! Lol maybe next year but like you said mostly everything else works, thanks for checking into it as well.
